.github/workflows/release.yml

```diff
@@ -291,17 +291,20 @@ jobs:
         with:
           name: image-replacements.txt
           path: ./image-replacements.txt
-      - name: Create portable coordinator resource definitions
+      - name: Create coordinator resource definitions
         run: |
           mkdir -p workspace
-          nix run .#scripts.write-coordinator-yaml -- "${coordinatorImgTagged}" > workspace/coordinator.yml
-          nix shell .#contrast --command resourcegen --image-replacements ./image-replacements.txt --platform AKS-CLH-SNP \
-            --namespace kube-system runtime > workspace/runtime.yml
-          nix shell .#contrast --command resourcegen --image-replacements ./image-replacements.txt --platform AKS-CLH-SNP \
-            --add-load-balancers emojivoto-sm-ingress > workspace/emojivoto-demo.yml
+          for platform in aks-clh-snp k3s-qemu-tdx k3s-qemu-snp rke2-qemu-tdx
+          do
+            nix run .#scripts.write-coordinator-yaml -- "${coordinatorImgTagged}" "${platform}" > workspace/coordinator-$platform.yml
+            nix shell .#contrast --command resourcegen --image-replacements ./image-replacements.txt --platform $platform \
+              --namespace kube-system runtime > workspace/runtime-$platform.yml
+            nix shell .#contrast --command resourcegen --image-replacements ./image-replacements.txt --platform $platform \
+              --add-load-balancers emojivoto-sm-ingress > workspace/emojivoto-demo-$platform.yml
```

Comment:
We should instead use underspecifided `contrast-cc` as runtime so we only need a single version of the deployment.